.golangci.yml

```diff
@@ -0,0 +1,29 @@
+# See https://github.com/golangci/golangci-lint#config-file
+run:
+  issues-exit-code: 1 #Default
+  tests: true #Default
+
+linters:
+  enable:
```

Comment:
is there plan to enable more linters? Also any reason for just this linters?

Reply:
This is on top of the default linters.